cryptus-app/components/market_overview/market_header/date/date.module.scss

```diff
@@ -0,0 +1,10 @@
+@use "/styles/variables" as v;
+
+.container {
+  flex: 1 1 auto;
+  color: #b7b7b7;
```

Comment:
use global

Reply:
What and where are globals ? @alexandrelafleur